### ISSUE TITLE
Cannot install Direwolf20 1.20 Pack

### ISSUE THREAD

**Issue by dragon-masterk** [FTBTeam/FTB-App#975]
### What Operating System

Windows 11

### Debug Code

FTB-DBGIWUPUKEDEV

### Describe the bug

Cannot install Direwolf 1.20 Pack

### Steps to reproduce

1. Click get button on the pack
2. Click install for the latest build, 1.0.01
3. Get a 'Fatal exception whilst preparing modpack instillation.'

### Expected behaviour

Download mods and install instance

### Screenshots

![image](https://github.com/FTBTeam/FTB-App/assets/83424830/1af1802f-bd73-484d-9ba3-35b9985d7e5d)

### Additional information

I can install and update other packs without any issues.

**Comment by Zalmun** [comment]
Same issue on Windows 10; haven't used the launcher since last year.

I was able to install the FTB University pack with no problems right afterwards though.

**Comment by Zalmun** [comment]
Just an update:  restarted overwolf and the FTB launcher after what I said above, then tried installing again, and it seems to install fine now.

**Comment by dragon-masterk** [comment]
So I tried everything, from restarting Overwolf and the FTB app, to restarting my PC. The only thing I did not do was reinstall the app, but I wasn't sure if that would preserve my save data on reinstall or if it would break things. This is why I preferred to just play any FTB packs with Prism Launcher instead of the broke FTB app. 

Version info:
UI Version: e5790011d3
App Version: 202208131844-e5790011d3-release
